compiler/qsc/src/interpret/stateful.rs

```diff
@@ -358,13 +357,23 @@ impl Interpreter {
     ) -> Result<Value, Vec<LineError>> {
         let mut result = Value::unit();
 
-        let mut fragments = self.compiler.compile_fragments(line).map_err(|errors| {
-            let source = line.into();
-            errors
-                .into_iter()
-                .map(|error| LineError(WithSource::new(Arc::clone(&source), error.into(), None)))
-                .collect::<Vec<_>>()
-        })?;
+        let line_label = format!("line_{}", self.lines);
+        self.lines += 1;
+
+        let mut fragments =
```

Comment:
This is where we "name" each line as we append it to the source map. This name can get surfaced later in error messages, such as:

```
Error: empty range
Call stack:
    at Microsoft.Quantum.Random.DrawRandomInt in random.qs
    at Main in <expression>
Qsc.Eval.EmptyRange

  × runtime error
  ╰─▶ empty range
   ╭─[line_0:1:1]
 1 │ operation Main() : Unit { Microsoft.Quantum.Random.DrawRandomInt(2,1); }
   ·                                                                 ──┬──
   ·                                                                   ╰── the range cannot be empty
   ╰────
```